mobile/src/main/java/butter/droid/ui/main/MainActivity.java

```diff
@@ -82,24 +86,31 @@
     @Inject ProviderManager providerManager;
 
     @BindView(R.id.toolbar) Toolbar mToolbar;
```

Comment:
Change the variable name here also removing the m suffix (as the others has been changed)